istioctl/pkg/writer/ztunnel/configdump/workload.go

```diff
@@ -129,25 +129,29 @@ func (c *ConfigWriter) PrintWorkloadDump(filter WorkloadFilter, outputFormat str
 	return nil
 }
 
-func waypointName(wl *ZtunnelWorkload, services map[string]*ZtunnelService) string {
+func waypointName(wl *ZtunnelWorkload, services []*ZtunnelService) string {
 	if wl.Waypoint == nil {
 		return "None"
 	}
 
-	if svc, ok := services[wl.Waypoint.Destination]; ok {
-		return svc.Name
+	for i := 0; i < len(services); i++ {
```

Comment:
not super important but could use `slices.FindFunc()` if you prefer. Or `for _, svc := range services`